apps/staging_sync/cli.py

```diff
@@ -0,0 +1,265 @@
+import datetime as dt
+from pathlib import Path
+from typing import Optional, Set
+
+import click
+import pandas as pd
+import structlog
+from dotenv import dotenv_values
+from rich import print
+from rich_click.rich_command import RichCommand
+from sqlalchemy.exc import IntegrityError, NoResultFound
+from sqlmodel import Session
+
+from etl import grapher_model as gm
+from etl.config import GRAPHER_USER_ID
+from etl.db import Engine, get_engine
+
+from .admin_api import AdminAPI
+
+log = structlog.get_logger()
+
+
+@click.command(cls=RichCommand, help=__doc__)
+@click.argument("source", type=Path)
+@click.argument("target", type=Path)
+@click.option(
+    "--chart-id",
+    type=int,
+    help="Sync only this chart id.",
+)
+@click.option(
+    "--publish/--no-publish",
+    default=False,
+    type=bool,
+    help="Automatically publish new charts.",
+)
+@click.option(
+    "--apply-revisions/--no-apply-revisions",
+    default=False,
+    type=bool,
+    help="""Directly update existing charts with approved revisions
+    (skip chart revision). Useful for large updates. This still
+    creates a chart revision if the target chart has been modified.""",
+)
+@click.option(
+    "--dry-run/--no-dry-run",
+    default=False,
+    type=bool,
+    help="Do not write to target database.",
+)
+def cli(
+    source: Path,
+    target: Path,
+    chart_id: Optional[int],
+    publish: bool,
+    apply_revisions: bool,
+    dry_run: bool,
+) -> None:
+    """Syncs grapher charts and revisions modified by Admin user from source_env to target_env. (Admin user is used by staging servers).
+
+    SOURCE and TARGET can be either paths to .env file or name of a staging server.
```

Comment:
Would it be possible to use here `master` as a target?

Reply:
Using `master` would sync them to `staging-sync-master` (which exists and is being cloned to new staging servers).

Writing to production shouldn't be too easy. It's ok for super users, but we might build some simple UI for others with unique MySQL user that can only write to selected tables.